From aaaf44d240aebd0eb3539b6e105a19a984000d81 Mon Sep 17 00:00:00 2001
From: Hans Meine <hans_meine@gmx.net>
Date: Fri, 4 Sep 2020 12:22:27 +0200
Subject: [PATCH] remove unnecessary parentheses (GCC9 warning)

---
 include/vigra/static_assert.hxx | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/vigra/static_assert.hxx b/include/vigra/static_assert.hxx
index a67390922..fffb8ae33 100644
--- a/include/vigra/static_assert.hxx
+++ b/include/vigra/static_assert.hxx
@@ -84,9 +84,8 @@ struct success {};
 inline int check( success ) { return 0; }
 
 template< typename Predicate >
-failure ************ (Predicate::************ 
-      assertImpl( void (*)(Predicate), typename Predicate::not_type )
-    );
+failure ************ Predicate::************ 
+      assertImpl( void (*)(Predicate), typename Predicate::not_type );
 
 template< typename Predicate >
 success
