From 68cfa41532835316835b88b7258ef93cb88234a5 Mon Sep 17 00:00:00 2001
From: Ullrich Koethe <ullrich.koethe@iwr.uni-heidelberg.de>
Date: Tue, 16 Oct 2018 20:23:15 +0200
Subject: [PATCH] cmake: more informative error message

---
 config/FindVIGRANUMPY_DEPENDENCIES.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/FindVIGRANUMPY_DEPENDENCIES.cmake b/config/FindVIGRANUMPY_DEPENDENCIES.cmake
index 47cf4237c..f64a52d72 100644
--- a/config/FindVIGRANUMPY_DEPENDENCIES.cmake
+++ b/config/FindVIGRANUMPY_DEPENDENCIES.cmake
@@ -80,7 +80,7 @@ IF(PYTHONINTERP_FOUND)
     IF(PYTHON_LIBRARIES)
         MESSAGE(STATUS "Found Python library: ${PYTHON_LIBRARIES}")
     ELSE()
-        MESSAGE(STATUS "Could NOT find Python library")
+        MESSAGE(STATUS "Could NOT find Python library (looking for version ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})")
     ENDIF()
 
     ######################################################################
@@ -130,7 +130,7 @@ IF(PYTHONINTERP_FOUND)
     if(Boost_PYTHON_LIBRARY)
         MESSAGE(STATUS "Found boost_python library: ${Boost_PYTHON_LIBRARY}")
     else()
-        MESSAGE(STATUS "Could NOT find boost_python library")
+        MESSAGE(STATUS "Could NOT find boost_python library (looking for version ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})")
     endif()
 
     ######################################################################
